### PR TITLE
PR for HPC training material

### PR DESCRIPTION
**Pull request by manasvita**
Initial commit for all files with slight modification.

### CODE REVIEW

**Comment by manasvita** [comment]
@joseph-cooke, sorry for this last minute notice but need it for my 9 AM session. Thanks.

File: Exercise_2/interactive_commad_list [hunk 1]
```diff
@@ -1,7 +1,11 @@
-cd $HOME/HPC_optimization_and_scalability
+cd /scratch/$USER/training-hpc 
+
 cd Exercise_2
+
 cat main_py.bash
+
 cat main_py.bash
```
Comment (joseph-cooke) [inline Exercise_2/interactive_commad_list:7]: This should probably be `cat calcMatrix.py`

**Review by joseph-cooke — APPROVED** [review]
Sorry for the longish time to review. I am still waiting on Exercise_3/weak_scaling/Exercise_2 to finish running. It has been running for over an hour and half. I topped into the node and it is running the python code it is just take a long time. You might want to change some of the parameters of that script unless you already have the data to do the analysis in the training. The same with Exercise_4/weak_scaling/Exercise_2, it is still in the queue and waiting to run. 

Everything else ran fine. Thank you!!

File: Exercise_2/interactive_commad_list (hunk 1 above)
Comment (manasvita) [reply Exercise_2/interactive_commad_list:7]: Thanks for catching that @joseph-cooke! Done.

**Comment by manasvita** [comment]
Thanks for the prompt review @joseph-cooke! For Exercise_3 & Exercise_4, I'm going to present them as additional exercises as there isn't much time to change anything. The reservation partition hasn't been used, so all exercises will run on regular partitions.

**Pull request merged**